com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.Prepass.cs

```diff
@@ -52,6 +52,12 @@ bool NeedClearGBuffer()
 
         HDUtils.PackedMipChainInfo GetDepthBufferMipChainInfo()
         {
+            // Initialize the MipChainInfo if nececcary. This will be necessary when rendering the first frame.
+            if (m_DepthBufferMipChainInfo.textureSize == Vector2Int.zero)
+            {
+                m_DepthBufferMipChainInfo.ComputePackedMipChainInfo(new Vector2Int(RTHandles.maxWidth, RTHandles.maxHeight));
```

Comment:
The problem here is that m_DepthBufferMipChainInfo was initialized during RT alloc with a callback, which has not been called yet during the render-graph setup of the first frame. So any pass that was using this was getting zeros (uninitialized data).
I'm not very familiar yet with RenderGraph, so I'm not sure if there is a better fix. 
@JulienIgnace-Unity If you can think something better please let me know.

Reply:
Question for @JulienIgnace-Unity , shouldn't the callback be called upon first usage and hence allocation of the RT?

Reply:
An alternative fix is to move all calls to GetDepthBufferMipChainInfo() inside the render callbacks/functions (so RT alloc will happen first). However it would be easy to re-introduce the first-frame bug if someone calls GetDepthBufferMipChainInfo() outside of a render callback in the future. That's why I opted for the fix that you see.

Reply:
@FrancescoC-unity yes it will but only when executing the graph. So any code relying on those values during graph construction will have wrong values.
Also this fix could lead to some more errors. It uses the global RTHandles system which might not always be the one for this texture. If we move this to history buffers for example this will break.
To me it seems that it would be easier to just call ComputePackedMipChainInfo with the HDCamera actualWidth/Height like it's done currently but at the beginning of ExecuteWithRenderGraph.
This way it should always work properly.

Reply:
> To me it seems that it would be easier to just call ComputePackedMipChainInfo with the HDCamera actualWidth/Height like it's done currently but at the beginning of ExecuteWithRenderGraph.
> This way it should always work properly.

Ok thanks, I will do that, test it, and push an update.

Reply:
And yes, this suggestion has worked. Thanks!